module-5/index.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Express } from 'express';
+import groupsRouter from './src/controllers/groups.controller';
+import usersRouter from './src/controllers/users.controller';
+import { AppDataSource } from './src/data-access/data-source';
+import logger from './src/services/logger.service';
+
+const app: Express = express();
+
+AppDataSource.initialize().then(() => {
+    app.use(express.json());
+    app.use('/users', usersRouter);
+    app.use('/groups', groupsRouter);
+
+    app.use((err, req, res, next) => {
+        logger.error(err);
+        if (err && err.error && err.error.isJoi) {
```

Comment:
Express determine middlewares with 4 params as error handler by default. No need to check `err`